conf/app.ini

```diff
@@ -439,6 +439,11 @@ SCHEDULE = @every 24h
 ; Archives created more than OLDER_THAN ago are subject to deletion
 OLDER_THAN = 24h
 
+; Synchronize external user data
```

Comment:
Same information just like when authorizing that is already in existing code, I'm just adding user information updating/new user creation on regular interval, so I don't think this would be appropriate place to specify such information in configuration file